src/ts/stringutils.ts

```diff
@@ -32,6 +33,26 @@ export namespace StringUtils {
         .replace('ss', leftPadWithZeros(seconds, 2));
   }
 
+  export function secondsToText(totalSeconds: number): string {
```

Comment:
why isn't `secondsToTime` sufficient?

Reply:
That only displays the time in format `00:00:00`. For voice over that doesn't actually help users understand what those numbers mean so `secondsToText` displays the format in `00 hours 00 minutes 00 seconds`

Reply:
👍